.zenodo.json

```diff
@@ -40,7 +45,7 @@
     {
       "name": "Beyer, Ross",
       "affiliation": "SETI Institute and NASA Ames Research Center",
-      "orcid": "0000-0003-4503-3335"
+      "orcid": "https://orcid.org/0000-0003-4503-3335"
```

Comment:
The .zenodo.json file "standard" is a hot mess as far as I can tell.  I thought that "orcid" here was just supposed to be the id, where are you seeing that it should be a URL?

Reply:
An ORCID is the full thing only:

https://support.orcid.org/hc/en-us/articles/360006897674-Structure-of-the-ORCID-Identifier

> The ORCID iD is an https URI with a 16-digit number that is compatible with the ISO Standard (ISO 27729), also known as the International Standard Name Identifier (ISNI), e.g. https://orcid.org/0000-0001-2345-6789

Reply:
I appreciate that is what the orcid.org people think, but as far as I can tell, Zenodo just wants the number without the URL prefix in their .zenodo.json files (e.g. https://developers.zenodo.org/, https://github.com/nipy/nipype/blob/master/.zenodo.json, https://github.com/perakakis/CoVidAffect/blob/master/.zenodo.json).